build/simp-environment-skeleton.spec

```diff
@@ -161,30 +79,609 @@ cd -
 %attr(0750,-,-) %{prefix}/secondary/FakeCA/gencerts_nopass.sh
 %attr(0750,-,-) %{prefix}/secondary/FakeCA/usergen_nopass.sh
 
```

Comment:
Where is the upgrade logic that removes the old selinux policy if it is loaded?  It isn't in this PR or https://github.com/simp/simp-selinux-policy/pull/1